module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```diff
@@ -1470,17 +1468,14 @@ public function getMyProfile($patron)
         );
         $expirationDate = !empty($expirationDate) ?
             $this->displayDate((string)$expirationDate[0]) : null;
-
-        return [
-            'firstname' => (string)($firstname[0] ?? null),
-            'lastname' => (string)($lastname[0] ?? null),
-            'address1' => $address1,
-            'address2' => $address2,
-            'zip' => $zip,
-            'phone' => null,  // TODO: phone number support
-            'group' => null,
-            'expiration_date' => $expirationDate,
-        ];
+        return $this->createProfileArray(
```

Comment:
Should we restore this TODO comment for future reference?

```suggestion
        // TODO: add phone number support
        return $this->createProfileArray(
```